src/cargo/core/compiler/timings.rs

```diff
@@ -609,8 +609,59 @@ static HTML_TMPL: &str = r#"
   <title>Cargo Build Timings — {ROOTS}</title>
   <meta charset="utf-8">
 <style type="text/css">
+:root {
+  --error-text: #e80000;
+  --text: #000;
+  --background: #fff;
+  --h1-border-bottom: #c0c0c0;;
+  --table-box-shadow: rgba(0, 0, 0, 0.1);
+  --table-th: #d5dde5;
+  --table-th-background: #1b1e24;
+  --table-th-border-bottom: #9ea7af;
+  --table-th-border-right: #343a45;
+  --table-tr-border-top: #c1c3d1;
+  --table-tr-border-bottom: #c1c3d1;
+  --table-tr-odd-background: #ebebeb;
+  --table-td-background: #ffffff;
+  --table-td-border-right: #C1C3D1;
+  --canvas-background: #f7f7f7;
+  --axes: #303030;
+  --grid: #e6e6e6;
+  --block: #aa95e8;
+  --custom-build: #f0b165;
+  --not-custom-build: #95cce8;
+  --dep-line: #ddd ;
+  --dep-line-highlighted: #000;
+  --cpu: rgba(250, 119, 0, 0.2)
+}
+
+@media (prefers-color-scheme: dark) {
```

Comment:
Could we split the dark theme support to the other commit?

That is, the first commit would be theming the current CSS color in CSS variables, followed by adding a dark color scheme media qeury.